Bottom: 692b9e2fc3e636a232a73a70f2f55b8bb20045f6
Top:    6528c6bbb54b961c1d88354c6c20dfb9ba83de31
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-05 15:46:27 +0100

hid/gtk: Don't test for NOOP before calling gtk_widget_set_sensitive()

We were using a deprecated test macro anyway, GTK already does a NOOP
test inside gtk_widget_set_sensitive().


---

diff --git a/src/hid/gtk/gui-library-window.c b/src/hid/gtk/gui-library-window.c
index 2cda08e..c709dbc 100644
--- a/src/hid/gtk/gui-library-window.c
+++ b/src/hid/gtk/gui-library-window.c
@@ -474,10 +474,7 @@ library_window_callback_filter_entry_changed (GtkEditable * editable,
   sensitive =
     (g_ascii_strcasecmp (gtk_entry_get_text (library_window->entry_filter),
 			 "") != 0);
-  if (GTK_WIDGET_IS_SENSITIVE (button) != sensitive)
-    {
-      gtk_widget_set_sensitive (button, sensitive);
-    }
+  gtk_widget_set_sensitive (button, sensitive);
 
   /* Cancel any pending update of the footprint list filter */
   if (library_window->filter_timeout != 0)
